### PR TITLE
Add three new mods

### PR DESCRIPTION
**Pull request by Banane9**
it's updatin' time

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -2994,6 +2994,55 @@
                 }
             }
         },
+        "Banane9.StopDisappearingLocomotionMenu": {
```
Comment (ljoonal) [inline manifest.json:2997]: contains funky IL stuff, will need to review properly later.

File: manifest.json [hunk 2]
```diff
@@ -2994,6 +2994,55 @@
...
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosStopDisappearingLocomotionMenu/releases/download/v1.0.0/StopDisappearingLocomotionMenu.dll",
+                            "sha256": "A94AD9CA070321088B690CB29A82B627BA8AFB4BA84C7D7036C1E99A3C44F2DB"
+                        }
+                    ]
+                }
+            }
+        },
+        "Banane9.FlexibleContactsSort": {
```
Comment (ljoonal) [inline manifest.json:3020]: 👍

**Comment by Banane9** [comment]
extra mod 👀

File: manifest.json (hunk 1 above)
Comment (ljoonal) [reply manifest.json:2997]: Fukit it's probably fine, the IL includes some funny jump/call instructions but don't see how they would do anything malicious
seal
.

**Review by ljoonal — APPROVED** [review]

File: manifest.json (hunk 1 above)
Comment (Banane9) [reply manifest.json:2997]: It's just some extra if stuff in IL 👀

**Pull request merged**